app/helpers/i18n_helper.rb

```diff
@@ -47,6 +47,38 @@ def date_eras_gregorian(date)
     nil
   end
 
+  # Return the most accurate date from an array of dates using scoring algorithm below
```

Comment:
I would rather just change the description and preserve the ability to allow a `single date` to be passed. In theory, as is the case with other elements, the API could return a single date of the form { 'date' } and we should be able to handle this case transparently.